### DISCUSSION TITLE
FLOWS - Multiple input connectors

### DISCUSSION THREAD

**Discussion by ernst86913** [directus/directus#17390] [Archive]
Is there a way to have multiple input connectors to one operation? 

Example:
- Flow with conditional statement
- Based on condition different operations are executed
- Afterwards for both conditions we again want to execute same operations again... therefore having both branches linking to one operation to bring things together again....

**Comment by luciandavidescu** [comment]
Make use of "trigger flow" for this purpose – i.e. in the initial flow chain together multiple trigger flows, have each of them do their thing and then have them trigger a single new flow.

**Reply by ernst86913** [reply]
Its a workaround - yes. But then it’s not ONE process anymore… and I end up having 1000s of flows which you cannot manage anymore holistically.

**Reply by ernst86913** [reply]
PS - error analysis is a pain too…

**Reply by luciandavidescu** [reply]
Myself I'd probably do it this way anyway since having connectors tangled all over the board would be even more of a mess. 
Ways to organize the flows themselves may help, maybe indenting of flows to show subordination or whatnot. 
Right now even basic sorting doesn't seem to work, they're thrown around based on UUID.
"error analysis is a pain too…"
oooouch!
